cli/templatecreate.go

```diff
@@ -127,7 +127,7 @@ func templateCreate() *cobra.Command {
 	cmd.Flags().StringVarP(&directory, "directory", "d", currentDirectory, "Specify the directory to create from")
 	cmd.Flags().StringVarP(&provisioner, "test.provisioner", "", "terraform", "Customize the provisioner backend")
 	cmd.Flags().StringVarP(&parameterFile, "parameter-file", "", "", "Specify a file path with parameter values.")
-	cmd.Flags().DurationVarP(&maxTTL, "max-ttl", "", 168*time.Hour, "Specify a maximum TTL for worksapces created from this template.")
+	cmd.Flags().DurationVarP(&maxTTL, "max-ttl", "", 2*time.Hour, "Specify a maximum TTL for workspaces created from this template.")
```

Comment:
Do we display the max TTL in the UI anywhere? I'm concerned about setting this so low without a clear indicator to the user.

Reply:
That's fair!